examples/chat-3rd-party-OAuth/src/botpress.d.ts

```diff
@@ -680,7 +680,7 @@ declare module 'botpress/sdk' {
     type?: any
     timeoutNode?: string
     flow?: string
-  } & (NodeActions)
+  } & NodeActions
```

Comment:
what is this file? `examples/chat-3rd-party-OAuth/src/botpress.d.ts`

Can't we just delete the whole file instead?

Reply:
lol this was unintentionnally commited